torchci/clickhouse_queries/weekly_force_merge_stats/query.sql

```diff
@@ -116,22 +115,21 @@ results AS (
       1,
       0
     ) AS force_merge_with_failures,
-    CAST(time as DATE) AS date
+    IF(
+      {one_bucket: Bool},
+      'Overall',
+      formatDateTime(
+        DATE_TRUNC({granularity: String}, time),
+        '%Y-%m-%d'
+      )
+    ) AS bucket
   FROM
     merges_identifying_force_merges
   ORDER BY
-    date DESC
-),
-bucketed_counts AS (
+    time DESC
+) , bucketed_counts AS (
   SELECT
-    IF(
-      : one_bucket,
-      'Overall',
-      FORMAT_TIMESTAMP(
-        '%Y-%m-%d',
-        DATE_TRUNC(: granularity, date)
-      )
-    ) AS granularity_bucket,
```

Comment:
Had to move this part up because CH errors when one_bucket is true due to the date field not being in the group by clause or in an aggregate function (which shouldn't matter since it doesn't get evaluated and gets set to 'Overall' when one_bucket is true, which is kind of like an aggregate function)

I have filed a ticket about this